templates/blog/post_list.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
+
+{% block content %}
+  {% for post in post_list %}
+    <div class="post">
+      <p><b>Title</b> : <a href="{% url 'blog:post-detail' post.id %}">{{ post.title }}</a></p>
+      <p>{{ post.owner }} at {{ post.created_time }}</p>
+      <p><b>Content</b> : {{ post.content }}</p>
+      <p>{{ post.commentaries.all.count }} Comment{{ post.commentaries.all.count|pluralize }}</p>
+    </div>
+    <br>
+  {% endfor %}
```

Comment:
It seems like you're using `post.commentaries.all.count` to count the number of comments for each post. This approach can lead to a large number of database queries, which could slow down your page load times if you have many posts. This is a common performance issue known as the 'N+1 problem'. To avoid this, consider using Django's `annotate` function in your view to add a count of comments to each post before it's passed to the template. This way, you'll only need one additional query instead of one for each post.